### PR TITLE
new(mp) : New Dell OS10 SNMP Monitoring Procedure [EN]

### PR DESCRIPTION
**Pull request by psamecentreon**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [X] 20.04.x
- [X] 20.10.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+id: network-dell-os10-snmp
+title: Dell OS10 SNMP
+---
+
+## Overview
+
+Dell OS10 is the new operating system for Dell network equipment. It combines the best of Linux, open computing and networking to advance open networking disaggregation.
+This Plugin-Pack monitors all physical and system points using SNMP.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:9]: You can simply remove this part, Plugin-Pack Assets / Monitored Objects is enough

File: en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+* Par intances :
+| Metric name                   | Description                               | Unit |
+|:----------------------------- |:----------------------------------------- | :--- |
+| os10CardStatus                | Status of the card                        |      |
+| os10FanOperStatus             | Status of the fan                         |      |
+| os10FanTrayOperStatus         | Status of the fantray                     |      |
+| os10PowerSupplyOperStatus     | Status of the power supply                |      |
+| resource.oper_status          | Status of the resources					|      |
+| hardware.temperature.celsius  | temperature of the different sensors      | C    |
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:70]: weird names and wrong indentation

File: en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+|:---------------------------------------- |:------------------------------------------------------- | :--- |
+| status                                   | Status of the interface                                 |      |
+| interface.traffic.in.bitspersecond       | Incoming traffic going through the interface.           | b/s  |
+| interface.traffic.out.bitspersecond      | Outgoing traffic going through the interface.           | b/s  |
+| interface.packets.in.error.percentage    | Incoming errored packets going through the interface.   | %    |
+| interface.packets.in.discard.percentage  | Incoming discarded packets going through the interface. | %    |
+| interface.packets.out.error.percentage   | Outgoing errored packets going through the interface.   | %    |
+| interface.packets.out.discard.percentage | Outgoing discarded packets going through the interface. | %    |
+
+It is possible to filter on the name of an interface using a REGEXP of the form [```--interface='^ens160$' --name``].
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:58]: Missing `

File: en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+| :-------------------------- | :------------ | :--- |
+| swap.usage.bytes            | Swap usage    | B    |
+| swap.free.bytes             | Swap free     | B    |
+| swap.usage.percentage       | Swap usage    | %    |
+
+<!--Uptime-->
+
+| Metric name                 | Description                                        | Unit |
+| :-------------------------- | :------------------------------------------------- | :--- |
+| uptime                      | Duration of system has been working and available. | s    |
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:103]: I think a nlabel exists for this metric

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 1 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Done

File: en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 3 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Added `

File: en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 4 above)
Comment (psamecentreon) [reply en/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Replaced with the correct label

**Review by Sims24 — APPROVED** [review]

**Pull request merged**